hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieFileInputFormatBase.java

```diff
@@ -317,4 +301,80 @@ private static RealtimeFileStatus createRealtimeFileStatusUnchecked(HoodieLogFil
 
     return targetFiles;
   }
+
+  private void validate(List<FileStatus> targetFiles, List<FileStatus> legacyFileStatuses) {
```

Comment:
Are the methods below merely moved?  Looks like the order changes.

Reply:
Yeah, GJF rule is applied

(non-static methods)
public 
protected 
private

(static methods)
public 
protected 
private